Task 4/script.js

```diff
@@ -0,0 +1,42 @@
+/* ------------------------------ TASK 4 -----------------------------------
+Parašykite JS kodą, vartotojui atėjus į tinklapį kreipsis į cars.json failą
+ir iš atvaizduos visus automobilių gamintojus ir pagamintus modelius. 
+Kiekvienas gamintojas turės savo atvaizdavimo "kortelę", kurioje bus 
+nurodomas gamintojas ir jo pagaminti modeliai.
+
+
+Pastaba: Sukurta kortelė, kurioje yra informacija apie automobilį (brand), turi 
+turėti bent minimalų stilių ir būti responsive;
+-------------------------------------------------------------------------- */
+
+const ENDPOINT = "cars.json";
+
+const users = async () => {
+  try {
+    const response = await fetch("./cars.json");
+    const data = await response.json();
+    displayResults(data);
+  } catch (error) {
+    console.log(error);
+  }
+};
+users();
+function displayResults(value) {
+  value.forEach(item => {
+    const card = document.createElement("div");
+
+    card.classList.add("card");
+
+    const brand = document.createElement("h2");
+
+    brand.textContent = item.brand;
+
+    const models = document.createElement("p");
+
+    models.textContent = item.models;
```

Comment:
Stiliaus ir responsyvumo nėra net minimalaus. Šioje vietoje truputį galėjote įdėti daugiau pastangų. Models yra array, kuri tiesiai dedate į paragrafo vidų, kas nėra sveikintina praktika, nes by default visos vertės yra sujungiamos per kabelelį be tarpo, kas tampa kaip vienas labai ilgas žodis, dėl ko ir responsyvumas dar labiau nukenčia. Galėjote čia panaudoti array join(", ") metodą, kas sujungtų visus modelius tvarkingai per kablelį su tarpais.